cmd/run.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"maps"
+	"slices"
+	"sync"
+	"time"
+
+	"github.com/C-L-I-M/chapter-dong-dong/config"
+	"github.com/C-L-I-M/chapter-dong-dong/discord"
+	"github.com/C-L-I-M/chapter-dong-dong/scappers"
+	"github.com/bwmarrin/discordgo"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func run(cmd *cobra.Command, _ []string) {
+	cfg, err := config.Load()
+	cobra.CheckErr(err)
+
+	session, err := discordgo.New("Bot " + cfg.DiscordToken)
+	cobra.CheckErr(err)
+
+	channels, err := discord.LoadChannels(session, cfg.ServerId, cfg.StaticChannels)
+	cobra.CheckErr(err)
+
+	cobra.CheckErr(channels.ResolveConfigDiff(slices.Sorted(maps.Keys(cfg.Sagas))))
+
+	var wg sync.WaitGroup
```

Comment:
This is where the fun begins lmao

Reply:
This is technically useless because I never created a way for sub-goroutines to properly stop so the `wg.Done()` will never get called so `wg.Wait()` will never finish except on SIGINT and SIGTERM of course (which is enough)